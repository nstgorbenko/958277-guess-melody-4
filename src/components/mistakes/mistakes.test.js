import Mistakes from "./mistakes.jsx";

import React from "react";
import renderer from "react-test-renderer";

describe(`Mistakes Component rendering`, () => {
  it(`renders correctly with 0 count`, () => {
    const tree = renderer
      .create(
          <Mistakes
            count={0}
          />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`renders correctly with 1 count`, () => {
    const tree = renderer
      .create(
          <Mistakes
            count={1}
          />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
