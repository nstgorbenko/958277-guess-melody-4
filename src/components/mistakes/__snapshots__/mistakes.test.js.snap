// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mistakes Component rendering Mistakes Component should render correctly with 0 count 1`] = `
<div
  className="game__mistakes"
/>
`;

exports[`Mistakes Component rendering Mistakes Component should render correctly with 1 count 1`] = `
<div
  className="game__mistakes"
/>
`;
